src/app/core/entity/model/entity.ts

```diff
@@ -196,6 +198,12 @@ export class Entity {
   })
   inactive: boolean;
 
+  /**
+   * Whether this entity has been anonymized and therefore cannot be re-activated.
+   */
+  @DatabaseField()
+  anonymized: boolean;
```

Comment:
Anonymize will often break the `toString` method. Should we add a fallback like `Anonymized child`? I think confluence does something like this when a user is deleted but still referenced in some docs.

Reply:
great suggestion! Will implement a fallback